packages/base-styles/_colors.scss

```diff
@@ -24,3 +24,8 @@ $light-gray-placeholder: rgba($white, 0.65);
 $alert-yellow: #f0b849;
 $alert-red: #cc1818;
 $alert-green: #4ab866;
+
```

Comment:
not sure if this is the best place to put these - went with CSS vars instead of SCSS vars so themes can potentially override these.

Reply:
The choice of CSS vars over SCSS makes sense to me 👍